src/javascript/jahia.js

```diff
@@ -1,5 +1,6 @@
 // eslint-disable-next-line no-undef, camelcase
 __webpack_public_path__ = `${window.contextJsParameters.contextPath}/modules/app-shell/javascript/apps/`;
+window.jahiaCommons.p = `${window.contextJsParameters.contextPath}/modules/app-shell/javascript/commons/`;
```

Comment:
what does it mean "P"

Reply:
a dirty things
a very very dirty things
but I guess it's for public path
but the minified version of it